Dockerfile.ocp

```diff
@@ -3,9 +3,10 @@ FROM ubi8
 # We don't need the deps that rhosp-director-images-ipa pulls in
 # use rpm to install without them to keep the image size down as
 # much as possible
-RUN yum install -y rhosp-director-images-ipa-$(uname -m) --downloadonly --downloaddir=/tmp/packages && \
-    rpm -i --nodeps /tmp/packages/rhosp-release-* /tmp/packages/rhosp-director-images-ipa-* && \
```

Comment:
Why not just remove rhosp-release from here? You use `--nodeps` anyway?

Reply:
it's not a matter of removing the package, but of not requiring it at all
downstream the package is not tagged for rhaos so dnf can't find it, and we don't want to tag it because we don't really need it

Reply:
Have you talked to someone downstream about why this dependency even exist? I don't see any sense in it.

Reply:
I talked to someone downstream about this dependency when we were tagging rhosp-director-images-ipa and the verdict was that we didn't need rhosp-release to be tagged (so not installable outside of openstack repos, and we're getting rid of them) considering that we only need the tarball inside the package in the process we run using get-resource.sh

Reply:
Yeah, but nor should the IPA images *ever* depend on rhosp-release. Can we just fix it downstream?

Reply:
I believe the dependency was added because of the tool contained in the rhosp-release package.
We could definitely ask for more insight on that, but in the meantime if we move forward with this we should be able to remove the last OSP repos depencendy.